src/NetMQ/ThreadSafeSocket.cs

```diff
@@ -63,7 +77,7 @@ public interface IRoutingIdSocket : IThreadSafeSocket
     /// </remarks>
     public abstract class ThreadSafeSocket: IThreadSafeSocket, IDisposable  
     {
-        private readonly SocketBase m_socketHandle;
+        internal readonly SocketBase m_socketHandle;
```

Comment:
```suggestion
        internal protected readonly SocketBase m_socketHandle;
```

Reply:
It won't compile, SocketBase is an internal class.